client/src/main/java/no/nav/common/client/norg2/CachedNorg2Client.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.common.client.norg2;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import no.nav.common.health.HealthCheckResult;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static no.nav.common.client.utils.CacheUtils.tryCacheFirst;
+
+public class CachedNorg2Client implements Norg2Client {
+
+    private final Norg2Client norg2Client;
+
+    private final Cache<String, List<Enhet>> alleAktiveEnheterCache = Caffeine.newBuilder()
```

Comment:
Skal jeg tolke dette dithen at vi skal bytte ut `ehcache` for `Caffeine`? Tenker vi ikke har behov for begge liksom. :P

Reply:
Kommer litt an på bruken. I common så tenker jeg caffeine er bedre siden det er lettere å bruke og mer lightweight enn ehcache. I apper så kan folk bruke litt hva de vil.

Reply:
Ta gjerne en liten sjekk med register-gjengen på hvorvidt de ønsker trafikk mot norg2 eller axsys også. Om de har samme infoen der da vel og merke. :P